app/controllers/spree/retail/shopify/hooks/refunds_controller.rb

```diff
@@ -0,0 +1,24 @@
+module Spree
+  module Retail
+    module Shopify
+      module Hooks
+        class RefundsController < HooksController
+          def create
```

Comment:
It's more of a Murphy's Law kind of thing. Maybe the load-balancer accidentally sends the request to two web servers, and the webhook action ends up getting invoked twice, completely by accident.

I might be thinking _too_ defensively here, though. What do you think?